mex/common/models/resource.py

```diff
@@ -118,10 +120,10 @@ class BaseResource(BaseModel):
     )
     temporal: list[Timestamp | str] = []
     theme: list[Theme] = Field(
```

Comment:
question: do we need to re-formulate the examples of any list to the inner type, to be inline with e.g. geprisId? or does that look the same in the generated schema anyway

```python
theme: list[
        Annotated[Theme, Field(..., examples=["https://mex.rki.de/item/theme-1"])]
    ] = Field(..., min_length=1)
```

Reply:
I checked, the schemas differ:

Giving the example for the entire type: `theme: list[str] = Field(examples=["https://mex.rki.de/item/theme-1"])`
```python
{'properties': {'theme': {'examples': ['https://mex.rki.de/item/theme-1'],
                          'items': {'type': 'string'},
                          'title': 'Theme',
                          'type': 'array'}},
}
```

Giving the example for the inner type: `theme: list[Annotated[str, Field(examples=["https://mex.rki.de/item/theme-1"])]]`
```python
{'properties': {'theme': {'items': {'examples': ['https://mex.rki.de/item/theme-1'],
                                    'type': 'string'},
                          'title': 'Theme',
                          'type': 'array'}},
}
```

I will have a quick look how much work this is and do it / write a ticket depending on the amount of required changes.

Reply:
done in 12ac412e97382fc7b73cfa12626101e888d6488e